packages/manager/modules/pci/src/projects/project/storages/object-storage/objects/onboarding/onboarding.controller.js

```diff
@@ -1,33 +1,41 @@
-import reduce from 'lodash/reduce';
+import { map } from 'lodash';
 import illustration from './assets/objects.png';
 
-import { GUIDES } from './onboarding.constants';
+import { GUIDES, STANDARD_S3 } from './onboarding.constants';
 
 export default class PciStorageObjectsOnboardingController {
   /* @ngInject */
-  constructor($translate, atInternet) {
+  constructor($translate, atInternet, coreConfig, ovhFeatureFlipping) {
     this.$translate = $translate;
     this.atInternet = atInternet;
+    this.user = coreConfig.getUser();
+    this.ovhFeatureFlipping = ovhFeatureFlipping;
   }
 
   $onInit() {
-    this.illustration = illustration;
-    this.guides = reduce(
-      GUIDES,
-      (list, guide) => [
-        ...list,
-        {
-          ...guide,
-          title: this.$translate.instant(
-            `pci_projects_project_storages_objects_onboarding_guides_${guide.id}_title`,
-          ),
-          description: this.$translate.instant(
-            `pci_projects_project_storages_objects_onboarding_guides_${guide.id}_description`,
-          ),
-        },
-      ],
-      [],
-    );
+    const standardS3offer = 'object-storage:standard-s3';
+    this.ovhFeatureFlipping
+      .checkFeatureAvailability(standardS3offer)
+      .then((newRegionFeature) =>
+        newRegionFeature.isFeatureAvailable(standardS3offer),
+      )
+      .then((status) => {
+        this.isS3User = status;
+        this.guideList = this.isS3User ? STANDARD_S3 : GUIDES;
+        this.illustration = illustration;
+        this.guides = map(this.guideList, (guide) => {
```

Comment:
you can use native map function:
```suggestion
        this.guides = this.guideList.map((guide) => {
```

Reply:
Okay Mohammed